TWLight/emails/tasks.py

```diff
@@ -481,19 +483,19 @@ def send_authorization_emails(sender, instance, **kwargs):
 def notify_applicants_when_waitlisted(sender, instance, **kwargs):
     """
     When Partners are switched to WAITLIST status, anyone with open applications
-    should be notified.
+    who hasn't already received an email for this application should be notified.
     """
     if instance.id:
         orig_partner = get_object_or_404(Partner, pk=instance.id)
 
-        if (
-            orig_partner.status != instance.status
-        ) and instance.status == Partner.WAITLIST:
-
+        status_changed = orig_partner.status != instance.status
```

Comment:
This change has nothing directly to do with the task, I just wanted to tidy up the superfluous brackets and clarify what was happening.